### ISSUE TITLE
bug(tests): local test is broken

### ISSUE THREAD

**Issue by martinobettucci** [scaleway/serverless-functions-node#64]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### What did you do?

Created a default node project:
```bash
$ export TEMPLATE=nodejs
$ serverless create \
  --path ${TEMPLATE}-func \
  --template-url https://github.com/scaleway/serverless-scaleway-functions/tree/master/examples/${TEMPLATE}
$ npm i @scaleway/serverless-functions
```

Used given exemple code:
```javascript

const url = require("url");

module.exports.handle = (event, context, callback) => {
  return {
    statusCode: 201,
    body: JSON.stringify({
      message: "Hello World!",
    }),
    headers: {
      "Content-Type": "application/json",
    },
  };
};

// This will execute when testing locally, but not when the function is launched
if ("file://" + __filename === url.pathToFileURL(process.argv[1]).href) {
  import("@scaleway/serverless-functions").then(scw_fnc_node => {
    scw_fnc_node.serveHandler(exports.handle, 8080);
  });
}
```

Run the local instance for testing:
```bash
$ node index.js
$ curl -X GET http://localhost:8080
> Hello World!
$ curl -X POST "http://127.0.0.1:8080" \
     -H "Content-Type: application/json" \
     -d '{
           "NfcCarte": "test001",
           "nfcID": "123159489",
           "lieu": "paris"
         }'

```

### What did you expect to see?

> Hello World!

### What did you see instead?

> Response code: 500 (Internal Server Error); Time: 5ms (5 ms); Content length: 238 bytes (238 B)

### What version of Node are you using (`node --version`)?

node20

```sh
$ node --version
```

### Does this issue reproduce with the latest release?

yes

**Comment by Shillaker** [comment]
Hi @martinobettucci I'm looking at this now. I notice in the setup that you're missing a `cd` (see 2nd command here):

```
$ serverless create \
  --path ${TEMPLATE}-func \
  --template-url https://github.com/scaleway/serverless-scaleway-functions/tree/master/examples/${TEMPLATE}
$ cd nodejs-func
$ npm i @scaleway/serverless-functions
```

Then for me the file is called `handler.js`. So after running the step above, I can run:

```
export NODE_ENV=test
node handler.js 
```

This starts a server in the foreground, which I can `curl` from another shell. Does this setup work for you?

**Comment by martinobettucci** [comment]
Indeed, the problem is with any other type of request instead of GET 😉

Yes, we possibly skipped some command line in the report and the lambda are compiled to index.js if you use ncc 🙏

So basically, only GET works local. Any other request like OPTION, POST, HEADER, PUT, DELETE it just fails with a server error.

Thank you for looking into it @Shillaker 🤩

**Comment by martinobettucci** [comment]
Any news about this? @Shillaker

**Comment by martinobettucci** [comment]
fixed in 1.1.1
thanks @Shillaker
